.github/workflows/create_docs.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
           ref: gh-pages
 
       - name: Set up Go 1.14
-        uses: actions/setup-go@v1
+        uses: actions/setup-go@v2
         with:
           go-version: 1.14
```

Comment:
`1.15` should be there too?

Reply:
that creates docs, so i think 1 version is enough, i can put the latest

Reply:
i pushed the latest @jonandernovella is that ok or should i still go with 2 versions ?